docs/01_introduction_and_goals.adoc

```diff
@@ -105,4 +105,5 @@ Table with role names, person names, and their expectations with respect to the
 | Students | Lorenzo La Rocca, Álvaro Rodríguez González, Pablo Lopez Amado, Andrés Martínez Rodríguez, Ángel Iglesias Préstamo, Noelia Iglesias Cuesta | Main developers of the application. They will develop the whole application and hence they will improve their teamwork capacity and coding skills.
```

Comment:
I would modify this line by adding an _and_ before Noelia's name instead of a _comma_.